src/activities/quizzes/QuizEntity.js

```diff
@@ -75,6 +75,13 @@ export class QuizEntity extends Entity {
 		return '';
 	}
 
+	outcomesTerm() {
+		if (!this._entity || !this._entity.properties) {
+			return;
+		}
+		return this._entity.properties['outcomes-term'];
```

Comment:
I see it here too: https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/0cfe543c0860d19c44e58cb5dd37a05a8bd76880/src/activities/assignments/AssignmentEntity.js#L35

Maybe centralize this string?